Charlie fly modifications (#3936)
freesewing/freesewing: markdown/org/docs/patterns/charlie/instructions/en.md

## Step 9:  Close the crossseam

> <tip>

Reviewer: ```suggestion
<Tip>
```

Reviewer: I believe this needs to be capitalized

Double check that your right and left legs match the pattern markings. The right leg's crotch should extend slightly beyond the left. Refer to the pattern and cut excess on the left leg if necessary.

> It can be valuable to mark with chalk or a running stitch the **Center Front** on each leg. This will help you make sure the peices are in the right place duing fly construction.

Reviewer: ```suggestion
It can be valuable to mark with chalk or thread mark the **Center Front** on each leg. This will help you make sure the peices are in the right place duing fly construction.
```

> </tip>

Reviewer: ```suggestion
</Tip>
```

Reviewer: I believe this needs to be capitalized

Make sure one leg is turned with the good side out, and the other has the good side in.

Now tuck the leg with the good side out inside the leg that has the good side in.
This way, they have their good sides against each other.

Align the cross seam, starting at the back waist, pinning both halves together as you make your way

> towards the fly. **Stop at the fly notch** and makes sure to back-stitch.

Reviewer: ```suggestion
towards the fly. **Stop at the fly notch** which is the bottom notch, making sure to back-stitch at the notch.
```

Feels best to indicate which notch this is in relevances to the others.

Author: In this case it's actually the top notch, but yes :)

When you're done. Do it again. Always sew the cross seam twice.
It's one of those best practices you ignore at your own peril.

> ![Close Crossseam](CloseCrossseam.svg)

Reviewer: ```suggestion
![Closing the Crossseam](CloseCrossseam.svg)
```

</Tip>

## Step 10: Construct the fly
### Close the fly extension and serge

Fold the fly extension on its fold line with good sides together.

Now sew along the bottom, to close the fly extension.

Trim back one half of the seam allowance to remove bulk before turning it good side out and pressing.

#### Serge the fly extension and fly facing

Serge (or zig-zag) along the open side of the fly extension. No need to serge the top as that will get caught in our waistband.

while you're at it, also serge along the entirety of the fly facings J-shaped edge.

> ![Close Extension and Serge](CloseExtensionAndSerge.svg)

Reviewer: ```suggestion
![Close the Extension and Serge the fly](CloseExtensionAndSerge.svg)
```

### Attach the fly facing

Sew fly facing to left leg. You will be sewing slightly past the cross seam point, so be careful to keep the right leg seam allowance out of the way and aim to sew directly on top of the cross seam line.

> ![Attach Facing](AttachFacing.svg)

Reviewer: ```suggestion
![Attaching the fly Facing](AttachFacing.svg)
```

### Finish the fly facing

Trim the fly facing to half of the seam allowance.	
Tuck in the tip of the fly facing and secure it with a few stitches.

> Edge stitch the fly facing to the seam allowance.

Reviewer: ```suggestion
_Edgestitch_ the fly facing to the seam allowance.
```

> Press the seam allowance toward the facing and understitch the facing for a clean finish. Again, keeping the right leg 	seam allowance out of the way.

Reviewer: ```suggestion
Press the seam allowance toward the facing and _understitch_ the facing for a clean finish. Again, keeping the right leg seam allowance out of the way.
```

> ![Finish Facing](FinishFacing.svg)

Reviewer: ```suggestion
![Finishing the Fly Facing](FinishFacing.svg)
```

> ### Edge stitch the zipper to the fly extension

Reviewer: ```suggestion
### Edgestitch the zipper to the fly extension
```

> <tip>

Reviewer: ```suggestion
<Tip>
```

**Zipper Length**

> It's a good idea to ensure your zipper stop ends before the indicated stitch line on the pattern. This will ensure you don't sew over your zipper stop and reduces tension on the bottom of the fly. If your zipper is too long, you can do a 0 length zig zag stitch to create a new stop (on just the zipper itself).

Reviewer: ```suggestion
It's a good idea to ensure your zipper stop ends before the indicated stitch line on the pattern. This will ensure you don't sew over your zipper stop and reduces tension on the bottom of the fly. If your zipper is too long, you can trim it down and create a new stopper by looping several stitches of thread inbetween the zipper teeth either by hand or using a 0 length zig zag stitch on your machine making sure to just sew on the zipper tape.
```

This may be more opinion. Either way maybe also adding a bit about using fray check or a dob of glue/superglue to secure the threads may be useful as well.

> </tip>

Reviewer: ```suggestion
</Tip>
```

> Edge stitch along the left edge of the zipper to the serged edge of the fly extension to hold it in place.

Reviewer: ```suggestion
_Edgestitch_ along the left edge of the zipper to the serged edge of the fly extension to hold it in place.
```

> ![Baste Zipper](BasteZipper.svg)

Reviewer: ```suggestion
![Baste the Zipper in place](BasteZipper.svg)
```

> ### Top stitch the right leg to the zipper

Reviewer: ```suggestion
### Topstitch the right leg to the zipper
```

Start by pressing the right leg seam allowance to the back.

> <tip>

Reviewer: ```suggestion
<Tip>
```

Remember that the seam allowance starts slightly right of the center front, the center front is marked in purple in the image, make sure you don't fold along the center front line by mistake.

> </tip>

Reviewer: ```suggestion
</Tip>
```

Ensure you place the folded edge of the right leg just left of the zipper teeth to prevent fabric from being caught in the zipper.

Stop sewing at the top 'fly extention' notch, slightly above the cross seam, and be sure to backstitch. If you don't do this you won't be able to keep the extension out of the way when sewing the J-Seam.

> ![Attach Extension to Right Leg](AttachExtensionToRightLeg.svg)

Reviewer: ```suggestion
![Attach the Extension to Right Leg](AttachExtensionToRightLeg.svg)
```

### Attach the zipper to the fly facing.

> Be careful to align the leg left on the right center front. It can be valuable to pin or baste through the facing and right zide of the zipper tape. (We need to sew the left side of the zipper tape to the left leg).

Reviewer: ```suggestion
Make sure to align the leg left to the right center front. It can be valuable to pin or baste through the facing and _good side_ of the zipper tape. (We need to sew the left side of the zipper tape to the left leg).
```

Author: good side isn't the intended meaning actually, I mean the zippers 'right side' like, right leg.

> Ensuring everything is lined, sew the fly facing to the left side of the zipper. Try to get close to the zipper, but not too close.

Reviewer: ```suggestion
Ensuring everything is aligned, sew the fly facing to the left side of the zipper. Try to get close to the zipper, but not too close.
```

Reviewer: Also "Try to get close to the zipper, but not too close." Is that saying to not close the zipper all the way? If so it may better to word it like. "Close the zipper leaving it slightly openg at the top"sor something like that thw wording is tricky so more of discussion.

Author: Hmm, I see your concern.. I've made a correction which is hopefully more clear? We sew the complete length of the zipper, we just don't want to sew too close to the zipper teeth to prevent the fabric from being caught

> ![Attach Facing to Zipper](AttachFacingToZipper.svg)

Reviewer: ```suggestion
![Attach the Facing to Zipper](AttachFacingToZipper.svg)
```

### Sew the J-seam of the fly

> <tip>

Reviewer: ```suggestion
<Tip>
```

Use a peice of paper as a guide to sew the J seam, this can be cut from your pattern along the indicated stitch line.

> </tip>

Reviewer: ```suggestion
</Tip>
```

> Now topstitch the so-called J-seam of the fly. Make sure to keep the fly extension out of the way. Since you didn't sew all the way along the fly extention, you should be able to pin it out of the way.

Reviewer: ```suggestion
Now _topstitch_ the so-called J-seam of the fly. Make sure to keep the fly extension out of the way. Since you didn't sew all the way along the fly extention, you should be able to pin it out of the way.
```

#### Bar tack the bottom of the fly

Now, bartack the very botton of the J-seam (the horizontal part) but this time make sure to also catch the fly extension.
You can choose to add an additional bar tack along the J-Seam as well (shown in purple).

> ![Sew the J-Seam](SewTheJSeam.svg)

Reviewer: ```suggestion
![Sewing the J-Seam](SewTheJSeam.svg)
```

Serge (or zigzag) the long edge of the strip on both sides.

> ![Finish Loop Edges](FinishLoopEdges.svg)

Reviewer: ```suggestion
![Finishing the Loop Edges](FinishLoopEdges.svg)
```
